src/html.cr

```diff
@@ -19,6 +19,15 @@ module HTML
       '\'' => "&#27;",
     },
   }
+  ESCAPE_JAVASCRIPT_SUBST = {
+    '\''     => "\\'",
+    '"'      => "\\\"",
+    '\\'     => "\\\\",
+    '\u2028' => "&#x2028;",
+    '\u2029' => "&#x2029;",
+    '\n'     => "\\n",
+    '\r'     => "\\n",
+  }
```

Comment:
django appears to escape slightly more? https://docs.djangoproject.com/en/1.10/_modules/django/utils/html/#escape Just asking... :)

Reply:
This PR has been closed and #5012 was merged instead. Currently there is no escape method for javascript.